exercises/01-Installing-Plugins/README.md

```diff
@@ -46,4 +48,13 @@ Navigate to http://localhost:8080/restart and click **Yes**, or check the box la
 
 Once Jenkins restarts, you'll be back to the sign in screen and you'll be able to use the new plugin.
 
+**Step 4: Install Other Course-Required Plugins**
+-------------------------------------------
+
+Now that you know how to install plugins, here's some of the other ones we're going to use during the course. Make sure these are installed before finishing the exercise.
+
+**SonarQube Scanner for Jenkins** 
+**GitHub Integration**
+**GitHub Branch Source**
```

Comment:
```suggestion
- **SonarQube Scanner for Jenkins** 
- **GitHub Integration**
- **GitHub Branch Source**
```

Don't know if you want a bulleted list, but these rendered all on one line, so recommending a list so they are all on seperate lines

Reply:
Also, maybe you can link to their page in the jenkins plugin directory, so we know we have the correct one